src/components/code-editor/code-editor.html

```diff
@@ -4,7 +4,7 @@
         <div class="modal-header" style="padding:0">
           <h2>{{ getFilename() }}</h2>
 	  <span style="position:absolute;top:0;right:0.2em;">
-	    <span v-if="isWritable" @click="getAndSave" style="color:black;font-size:2.5em;font-weight:bold;cursor:pointer;margin:.3em;" v-bind:class="['fas', saving ? 'fa-hourglass' : 'fa-save']" title="Save file"></span>
+	    <span v-if="isWritable()" @click="getAndSave" style="color:black;font-size:2.5em;font-weight:bold;cursor:pointer;margin:.3em;" v-bind:class="['fas', saving ? 'fa-hourglass' : 'fa-save']" title="Save file"></span>
```

Comment:
isWritable or isFileWritable?

Reply:
isWritable is the function in the JS file.